packages/terra-framework-docs/src/terra-dev-site/doc/menu/example/MenuWithCustomIcons.jsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable react/no-access-state-in-setstate */
+import React from 'react';
+import Button from 'terra-button';
+import PropTypes from 'prop-types';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconBookmark from 'terra-icon/lib/icon/IconBookmark';
+import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
+import IconEnvelope from 'terra-icon/lib/icon/IconEnvelope';
+import IconPrinter from 'terra-icon/lib/icon/IconPrinter';
+import classNames from 'classnames/bind';
+import styles from './BasicMenu.module.scss';
+import Menu from '../../../../../../terra-menu/lib/Menu';
```

Comment:
```suggestion
import Menu from 'terra-menu';
```
Does the above way of import caused any issues ??

Reply:
I had to reference it directly for developmental purposes, but is safe to set it back
https://github.com/cerner/terra-framework/pull/1592/commits/e1296db724232637ea84ce8577baa3536604c27f